App/UI/Settings/CustomerSupport/CustomerSupportView.swift

```diff
@@ -0,0 +1,309 @@
+// CustomerSupportView.swift
+// Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+// Please refer to the AUTHORS file for more information.
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import Extensions
+import Foundation
+import Tempura
+
+struct CustomerSupportVM: ViewModelWithLocalState {
+  enum CellType: Equatable {
+    case title(String)
+    case textualContent(String)
+    case button(String, String)
+    case separator
+    case spacer(ContentCollectionSpacerVM.Size)
+    case contact(CustomerSupportContactCellVM.Kind)
+  }
+
+  /// The array of cells in the collection.
+  let cells: [CellType]
+  /// Whether the header is visible in the view. The header is shown only when the content is scrolled.
+  let isHeaderVisible: Bool
+
+  func shouldReloadCollection(oldVM: Self?) -> Bool {
+    guard let oldVM = oldVM else {
+      return true
+    }
+
+    return self.cells != oldVM.cells
+  }
+
+  func shouldUpdateHeader(oldVM: Self?) -> Bool {
+    guard let oldVM = oldVM else {
+      return true
+    }
+
+    return self.isHeaderVisible != oldVM.isHeaderVisible
+  }
+
+  func cellVM(for item: CellType) -> ViewModel? {
+    switch item {
+    case .title(let title):
+      return ContentCollectionTitleCellVM(content: title)
+    case .textualContent(let content):
+      return ContentCollectionTextCellVM(content: content)
+    case .button(let description, let title):
+      return ContentCollectionButtonCellVM(description: description, buttonTitle: title)
+    case .separator:
+      return ContentCollectionImageCellVM(content: Asset.Common.separator.image)
+    case .spacer(let size):
+      return ContentCollectionSpacerVM(size: size)
+    case .contact(let kind):
+      return CustomerSupportContactCellVM(kind: kind)
+    }
+  }
+}
+
+extension CustomerSupportVM {
+  static let cells: [CellType] = [
```

Comment:
[i]be aware that we may not have the email (settings) or the phone / start / end time (settings).

Reply:
I know, also the `useful info` section will have some info computed for sure. 
Maybe I'll deprecate this.